qualtran/bloqs/factoring/ecc/_ec_add.py

```diff
@@ -0,0 +1,99 @@
+#  Copyright 2023 Google LLC
```

Comment:
What's going on with the module naming here? Why are they all private modules?

Reply:
I figured this would be an extra nudge to import from e.g. `qualtran.bloqs.factoring.ecc` rather than the specific module (and I'm keeping to a relatively small number of bloqs per module). I guess we don't do this anywhere else, though

Reply:
changed